docs/community/2-sdlc/code-contribution-guidelines/coding-standards/README.md

```diff
@@ -63,7 +63,7 @@ code easier to read. Ed-Fi code conforms to the following conventions:
 
    **C# Example**
 
-   ```csharp
+   ```cs
```

Comment:
`csharp` is actually correct, not `cs`, for codeblocks. Though `cs` _might_ work.